packages/@react-spectrum/list/docs/ListView.mdx

```diff
@@ -0,0 +1,345 @@
+      }
+
+      let res = await fetch(cursor || `https://swapi.py4e.com/api/people/?search=`, {signal});
+      let json = await res.json();
+
+      return {
+        items: json.results,
+        cursor: json.next
+      };
+    }
+  });
+
+  return (
+    <ListView
+        aria-label="example async loading list"
+        height="size-3000"
+        items={list.items}
+        loadingState={list.loadingState}
+        onLoadMore={list.loadMore}>
+        {(item) => (
+          <Item key={item.name} textValue={item.name}>{item.name}</Item>
+        )}
+    </ListView>
+  );
+}
+```
+
+## Complex items
+
+Items within a ListView also allow for additional content used to add context or provide additional actions to Items. Icons, links, and descriptions can be added to the children of Item as shown in the example below. If a description is added, the prop `slot="description"` must be used to distinguish the different `<Text>` elements. Additionally, components such as `<ActionButton>`, `<ActionGroup>`, and `<ActionMenu>` will be styled appropriately if included within an item.
+
+```tsx example
+<ListView aria-label="List example with complex items">
+  <Item textValue="Row with link, icon, and description">
+    <Folder />
+    <Link>Row with link, icon, and description</Link>
+    <Text slot="description">Description of row</Text>
+  </Item>
+  <Item textValue="Row with action button">
+    Row with action button
+    <ActionButton aria-label="Info">
+      <Info />
+    </ActionButton>
+  </Item>
+  <Item textValue="Row with action group">
+    Row with action group
+    <ActionGroup buttonLabelBehavior="hide">
+      <Item key="edit">
+        <Edit />
+        <Text>Edit</Text>
+      </Item>
+      <Item key="delete">
+        <Delete />
+        <Text>Delete</Text>
+      </Item>
+    </ActionGroup>
+  </Item>
+  <Item textValue="Row with action menu">
+    Row with action menu
+    <ActionMenu>
+      <Item key="edit" textValue="Edit">
+        <Edit />
+        <Text>Edit</Text>
+      </Item>
+      <Item key="delete" textValue="Delete">
+        <Delete />
+        <Text>Delete</Text>
+      </Item>
+    </ActionMenu>
+  </Item>
+</ListView>
+```
+
+## Selection
+
+By default, ListView doesn't allow row selection but this can be enabled using the `selectionMode` prop.
+Use `defaultSelectedKeys` to provide a default set of selected rows. Note that the value of the selected keys must match the `key` prop of the Item.
+
+The example below enables multiple selection mode, and uses `defaultSelectedKeys` to select the rows with keys "row1" and "row3".
+
+```tsx example
+<ListView selectionMode="multiple" defaultSelectedKeys={["row1", "row3"]} aria-label="List seletion example">
+  <Item key="row1" textValue="row 1">
+    <Content>row 1</Content>
+  </Item>
+  <Item key="row2" textValue="row 2">
+    <Content>row 2</Content>
+  </Item>
+  <Item key="row3" textValue="row 3">
+    <Content>row 3</Content>
+  </Item>
+</ListView>
```

Comment:
IMO, replace the values here w/ the same pokemon values as the examples below

Reply:
Good idea, fixed